spec/configuration.yml.example

```diff
@@ -7,11 +7,11 @@ root:
 user:
   host: localhost
   username: LOCALUSERNAME
-  password:
+  password: test
   database: mysql2_test
 
 numericuser:
   host: localhost
   username: LOCALUSERNAME
-  password:
+  password: test
```

Comment:
The user is implicitly created by this `GRANT`: https://github.com/brianmario/mysql2/pull/669/files#diff-f50f287044ea76fe2dfe616c8d01c194R203

We could explicitly `CREATE USER …` in spec helper's `config.before(:all)`, but that was added in MySQL 5.0.2.